R/rwebppl.R

```diff
@@ -343,21 +344,20 @@ run_webppl <- function(program_code = NULL, program_file = NULL, data = NULL,
   if (file.exists(error_file)) {
     file.remove(error_file)
   }
+  finish_file <- sprintf("/tmp/webppl_finished_%s", uid)
+  finish_arg <- finish_file
```

Comment:
I was doing this to mimic the code structure preceding it (with `output_file` and `output_arg`), but then, I'm not really sure why those are there either... maybe it's worth scrapping those lines too?